code/processes/rdb.q

```diff
@@ -30,6 +31,7 @@ parvaluesrc:@[value;`parvaluesrc;`log];						//where to source the rdb partition
                                                             //tab (from the the first value in the time column of the table that is subscribed for) 
                                                             //anything else will return a null date which is will be filled by pardefault									
 pardefault:@[value;`pardefault;.z.D];				        //if the src defined in parvaluesrc returns null, use this default date instead 
+tpcheckcycles:@[value;`tpcheckcycles;5];                    //specify the number of times the process will check for an available tickerplant
```

Comment:
This should be 0W to maintain backward functionality

Reply:
will change this after I am done with testing.